package.json

```diff
@@ -297,7 +297,8 @@
     "json5": "2.2.2",
     "nth-check": "2.0.1",
     "set-value": "4.0.1",
-    "unset-value": "2.0.1"
+    "unset-value": "2.0.1",
+    "webpack-dev-middleware": "^5.3.4"
```

Comment:
There was a conflict because `source-map-js` is in `master` but not in 49 branch.